README.md

```diff
@@ -1,24 +1,80 @@
+--------
+
+This crate provides unsafe `ffi` bindings in the `oqs-sys` crate, and safe wrappers are offered via the `oqs` crate.
+
+Usage
+-----
+
+<span style="color: red;">TODO</span>
+
+Running
+-------
+
+<span style="color: red;">TODO</span>
+
+Adding new algorithms
+---------------------
+
+### KEMs
 
 1. Update the Git submodule
-1. `oqs-sys` will now update when you build again
-1. Add it to the ``implement_kems!`` macro call in ``oqs/src/kem.rs``:
-  * The structure is a name for the algorithm in CamelCase, and the name of the constant of the algorithm (``OQS_KEM_alg_...``)
+2. `oqs-sys` will now update when you build again
+3. Add it to the ``implement_kems!`` macro call in ``oqs/src/kem.rs``:
+  - The structure is a name for the algorithm in CamelCase, and the name of the constant of the algorithm (``OQS_KEM_alg_...``)
 
 ### Signature schemes:
 
 1. Update the Git submodule
-1. `oqs-sys` is now up-to-date when you build again
-1. Add it to ``implement_sigs!`` macro call in ``oqs/src/sig.rs``.
-  * The structure is a name for the algorithm in CamelCase, and the name of the constant of the algorithm (``OQS_SIG_alg_...``)
+2. `oqs-sys` is now up-to-date when you build again
+3. Add it to ``implement_sigs!`` macro call in ``oqs/src/sig.rs``.
+  - The structure is a name for the algorithm in CamelCase, and the name of the constant of the algorithm (``OQS_SIG_alg_...``)
+
+Limitations and security
+------------------------
+
+liboqs is designed for prototyping and evaluating quantum-resistant cryptography. Security of proposed quantum-resistant algorithms may rapidly change as research advances, and may ultimately be completely insecure against either classical or quantum computers.
+
+We believe that the NIST Post-Quantum Cryptography standardization project is currently the best avenue to identifying potentially quantum-resistant algorithms. liboqs does not intend to "pick winners", and we strongly recommend that applications and protocols rely on the outcomes of the NIST standardization project when deploying post-quantum cryptography.
+
+We acknowledge that some parties may want to begin deploying post-quantum cryptography prior to the conclusion of the NIST standardization project. We strongly recommend that any attempts to do make use of so-called **hybrid cryptography**, in which post-quantum public-key algorithms are used alongside traditional public key algorithms (like RSA or elliptic curves) so that the solution is at least no less secure than existing traditional cryptography.
+
+Just like liboqs, liboqs-rust is provided "as is", without warranty of any kind. See [LICENSE.txt](https://github.com/open-quantum-safe/liboqs-rust/blob/master/LICENSE.txt) for the full disclaimer.
+
+License
+-------
+
+<span style="color: red;">TODO: Is this the correct license, Thom?</span>
+
+liboqs-rust is licensed under the MIT License; see [LICENSE.txt](https://github.com/open-quantum-safe/liboqs-rust/blob/master/LICENSE.txt) for details.
```

Comment:
The "common" licensing for Rust projects is to dual-license under both MIT and Apache 2, and is what I've used here. Removing Apache2 for consistency with the rest is fine with me.

Reply:
I've fine with including Apache 2, I just didn't know what you wanted, as I didn't see a LICENSE file in the repo.